pinot-query-planner/src/main/java/org/apache/pinot/query/catalog/PinotCatalog.java

```diff
@@ -44,14 +50,46 @@
  */
 public class PinotCatalog implements Schema {
 
+  public static final String DEFAULT_DB_NAME = "default";
```

Comment:
do we need to use `default` here? in postgres the default schema is `public`

Reply:
This decision was made as part of #12417 . This has some history related to the existing integrations that we have with integrations such as [superset](https://github.com/python-pinot-dbapi/pinot-dbapi/blob/master/pinotdb/sqlalchemy.py#L223), [presto](https://github.com/prestodb/presto/blob/master/presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotMetadata.java#L62), etc where we currently use `default` as the schema/database name placeholder and using any other name may break existing flows.